admin_manual/configuration_server/occ_command.rst

```diff
@@ -12,6 +12,16 @@ setting, and more.
 run it as your HTTP user** to ensure that the correct permissions are maintained
 on your Nextcloud files and directories.
 
+.. note:: 
+  Since APCu is disabled by default for the command-line mode of PHP, it can cause issues with Nextcloud’s ``occ`` command. Please make sure you set the ``apc.enable_cli`` parameter to ``1`` in your PHP CLI's ``php.ini`` configuration file or append ``--define apc.enable_cli=1`` each time you invoke ``occ`` - e.g.::
```

Comment:
wouldnt this only apply if apcu is actually enabled?

Reply:
Due to the better performance with APCu, most people have it enabled.

Is there a problem if the APCu module is enabled on CLI but not used because it's not enabled in Nextcloud?